### PR TITLE
Updating environment variables

### PR DESCRIPTION
**Pull request by ivana-companieshouse**

### CODE REVIEW

File: terraform/groups/ecs-service/locals.tf [hunk 1]
```diff
@@ -80,18 +67,17 @@ locals {
     { "name": "MONGODB_URL", "valueFrom": "${local.service_secrets_arn_map.mongodb_url}"},
     {"name": "STAFFWARE_DB_USER", "valueFrom": "${local.service_secrets_arn_map.staffware_db_user}"},
     {"name": "STAFFWARE_DB_PASSWORD", "valueFrom": "${local.service_secrets_arn_map.staffware_db_password}"}, 
-    {"name": "STAFFWARE_DB_CONNECTIONSTRING", "valueFrom": "${local.service_secrets_arn_map.staffware_db_connectionstring}"},       
+    {"name": "STAFFWARE_DB_CONNECTIONSTRING", "valueFrom": "${local.service_secrets_arn_map.staffware_db_connectionstring}"},
```
[the remark this answers is not in the document]
Comment (ivana-companieshouse) [reply terraform/groups/ecs-service/locals.tf:70]: I don't think so, but I'd keep it aligned with the other services

File: terraform/groups/ecs-service/locals.tf [hunk 2]
```diff
@@ -80,18 +67,17 @@ locals {
...
+    { "name": "COOKIE_SECRET", "valueFrom": "${local.secrets_arn_map.web-oauth2-cookie-secret}" },
+    { "name": "COOKIE_SERVER", "valueFrom": "${local.service_secrets_arn_map.cookie_server}" }   
   ]
 
   task_environment = [
-    { "name": "LOG_LEVEL", "value": "${var.log_level}" },
-    { "name": "CHS_URL", "value": "${var.chs_url}" },
     { "name": "CDN_HOST", "value": "//${var.cdn_host}" },
```
Comment (bcullerton) [inline terraform/groups/ecs-service/locals.tf:76]: The CDN_HOST in layout.html already has // prepended to the variable. I don't think it's needed here:

https://github.com/companieshouse/transaction-search-tool/blob/main/src/views/layout.html#L7

**Review by bcullerton — APPROVED** [review]

File: terraform/groups/ecs-service/locals.tf [hunk 3]
```diff
@@ -50,48 +42,42 @@ locals {
...
   staffware_db_connectionstring = local.service_secrets["staffware_db_connectionstring"]
-  mongodb_url                   = local.service_secrets["mongodb_url"]
 
   # create a map of secret name => secret arn to pass into ecs service module
   # using the trimprefix function to remove the prefixed path from the secret name
   secrets_arn_map = {
-    for sec in data.aws_ssm_parameter.secret:
-      trimprefix(sec.name, "/${local.name_prefix}/") => sec.arn
+    for sec in data.aws_ssm_parameter.secret :
+    trimprefix(sec.name, "/${local.name_prefix}/") => sec.arn
```
Comment (tobafolaranmi-ch) [inline terraform/groups/ecs-service/locals.tf:50]: Any reason for the tab space change here?

File: terraform/groups/ecs-service/locals.tf [hunk 4]
```diff
@@ -50,48 +42,42 @@ locals {
...
   }
 
   service_secrets_arn_map = {
-    for sec in module.secrets.secrets:
-      trimprefix(sec.name, "/${local.service_name}-${var.environment}/") => sec.arn
+    for sec in module.secrets.secrets :
+    trimprefix(sec.name, "/${local.service_name}-${var.environment}/") => sec.arn
```
Comment (tobafolaranmi-ch) [inline terraform/groups/ecs-service/locals.tf:55]: https://github.com/companieshouse/transaction-search-tool/pull/90/files#r1335548947

Comment (ivana-companieshouse) [reply terraform/groups/ecs-service/locals.tf:55]: Not really, updated automatically by the linter.

File: terraform/groups/ecs-service/locals.tf (hunk 3 above)
Comment (ivana-companieshouse) [reply terraform/groups/ecs-service/locals.tf:50]: Not really, updated automatically by the linter.

**Pull request merged**